Demo/Configuration.swift

```diff
@@ -26,7 +26,7 @@ internal enum ConfigurationConstants {
 
     static let returnUrl = "ui-host://payments"
     
-    static let shopperReference = "iOS Checkout Shopper"
+    static let shopperReference = "iOSCheckoutShopper"
```

Comment:
What is the reason for this?

Reply:
For UPI payments there is some validation for shopperReference  on billdesk. It does not take shopperReference  with spaces but this I committed by mistake I will revert this.

Reply:
Still need to revert this one

Reply:
yeah reverted it. 1b6fd0226a399eddb458267493bc95578d2df703